apps/webhook/src/webhooks/webhooks.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { Body, ClassSerializerInterceptor, Controller, Param, Post, UseInterceptors } from '@nestjs/common';
+import { Webhook, IWebhookResult } from './usecases/webhook/webhook.usecase';
+import { WebhookCommand } from './usecases/webhook/webhook.command';
+
+@Controller('/webhooks')
+@UseInterceptors(ClassSerializerInterceptor)
+export class WebhooksController {
+  constructor(private webhookUsecase: Webhook) {}
+
+  @Post('/:organizationId/:environmentId/email/:providerId')
```

Comment:
```suggestion
  @Post('/organizations/:organizationId/environments/:environmentId/email/:providerId')
```
I am a big advocate of following typically established REST patterns of resource naming even if it creates longer path names, as it provides flexibility and avoids URL clashing. But the most important thing would be to be consistent throughout the monolith and that might worth a bigger discussion out of this PR.